Source/WebCore/loader/FrameLoader.cpp

```diff
@@ -1751,6 +1751,9 @@ void FrameLoader::load(FrameLoadRequest&& request)
     loader->setIsContentRuleListRedirect(request.isContentRuleListRedirect());
     loader->setIsRequestFromClientOrUserInput(request.isRequestFromClientOrUserInput());
     loader->setIsContinuingLoadAfterProvisionalLoadStarted(request.shouldTreatAsContinuingLoad() == ShouldTreatAsContinuingLoad::YesAfterProvisionalLoadStarted);
+    if (crossSiteRequester)
+        loader->setCrossSiteRequester(*crossSiteRequester);
```

Comment:
if you manage to take in a r-value reference, please use `WTFMove(*crossSiteRequester)` here and update `setCrossSiteRequester` to take in a r-value reference and use `WTFMove()` internally.